lib/prawn/table/cell.rb

```diff
@@ -710,9 +710,9 @@ def border_left_line=(val)
       # Draws the cell's background color.
       #
       def draw_background(pt)
-        if defined?(@background_color) && @background_color
+        if (bg_color = background_color)
```

Comment:
To avoid calling the method a second time inside the conditional block. However, I realize this optimization is probably unnecessarily (and, in some cases, a deoptimization). I'll update it once I get the ping from @packetmonkey.